server-common/src/main/java/org/apache/cassandra/sidecar/common/server/MetricsOperations.java

```diff
@@ -32,4 +33,9 @@ public interface MetricsOperations
      */
     ConnectedClientStatsResponse connectedClientStats(boolean summaryOnly);
 
+    /**
+     * Retrieve the stream progress stats metrics from the cluster
+     * @return the requested stream progress stats, in full or summary
```

Comment:
since `in full or summary` is mentioned, can you explain what is full and what is summary in the javadoc.